examples/resources/opslevel_tag/resource.tf

```diff
@@ -0,0 +1,19 @@
+data "opslevel_repository" "foo" {
```

Comment:
Can we add an `import.sh` file in this directory for the tags resource?

Reply:
Last week @rocktavious mentioned something about us holding off on an `import.sh` for this one.
I think it had to do with us needing to define a custom `schema.ResourceImporter` since this is a special case?

Reply:
Yes - this resource is not easily importable because the customer would have to use a stanza like

```sh
terraform import Service:Z2lkOi8vMTIzNDU2Nzg5OTg3NjU0MzIx:Z2lkOi8vMTkyODM3NDY1NTY0NzM4Mjkx
```

or

```sh
terraform import InfrastructureResource:Z2lkOi8vMTIzNDU2Nzg5OTg3NjU0MzIx:Z2lkOi8vMTkyODM3NDY1NTY0NzM4Mjkx
```

and then yes we'll have to write a customer importer to handle the 3 peices of information we need

- TaggableResource
- ResourceID
- TagID

Its non-trivial AND customers likely will not need to do this so i'd rather wait until it becomes a huge need.